lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,22 +24,33 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
```

Comment:
(If you're unfamiliar with [`Array#map`](https://ruby-doc.org/core-2.2.0/Array.html#method-i-map) and [`Enumerable#each_with_object`](https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-each_with_object), I'd highly recommend reading about them and experimenting a bit)